tests/test_coordinates/test_transforms.py

```diff
@@ -0,0 +1,145 @@
+from celest.coordinates import LVLH
+from celest.coordinates.frames.gcrs import GCRS
+from celest.coordinates.ground_location import GroundLocation
+from celest.coordinates.frames.itrs import ITRS
+from celest.coordinates.frames.wgs84 import WGS84
+from celest.coordinates.transforms import (
+    _gcrs_to_itrs,
+    _itrs_to_gcrs,
+    _altitude,
+    _itrs_to_wgs84,
+    _wgs84_to_itrs,
+    _itrs_to_azel, _gcrs_to_lvlh
+)
+from celest import units as u
+from unittest import TestCase
+import numpy as np
+
+
+class TestCoordinateTransformations(TestCase):
+
+    def setUp(self):
```

Comment:
There are multiple test classes that have a very similar `setUp` method. Consider making a utilities file that allows you to initialize different coordinate frames using a function call.